nengo_gui/static/menu.js

```diff
@@ -27,6 +27,8 @@ Nengo.Menu.prototype.show = function (x, y, items) {
         return;
     }
 
+    this.menu_margin = 15;
+
```

Comment:
> This was in the case of it being full screen, it would simply spill over onto the taskbar.

But is there an equivalent to this ```menu_margin``` thing?  Does it keep a buffer of a fixed amount of minimum space between the menu and the farthest edge of the screen?